shared/scss/base/base.scss

```diff
@@ -392,3 +407,40 @@ a {
         }
     }
 }
+
+/* Popout box with left arrow */
+.modal-box {
```

Comment:
I thought this is a generic element that we might want to maybe reuse at some point, and as such it belongs more in the base styles than in the ones specifically for the whitelist